### PR TITLE
magento/devdocs#: Get IP of Cloud instance

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information how to get IP address of Cloud instance.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/cloud/env/environments-start.html

## Links to Magento source code

Thank you!

whatsnew
Added an example for retrieving an IP address for particular Cloud instance in the [Integration environment IP address section](https://devdocs.magento.com/guides/v2.2/cloud/env/environments-start.html#ipaddress) of the Cloud guide.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
I still wonder why don't people use `ping` for this purpose. Basically, there is a variety of options how to achieve that. But from my perspective, the `ping` operation is most basic since it's present on most desktop systems (even Windows) out of the box.

**Comment by atwixfirster** [comment]
> I still wonder why don't people use `ping` for this purpose. Basically, there is a variety of options how to achieve that. But from my perspective, the `ping` operation is most basic since it's present on most desktop systems (even Windows) out of the box.

The existing Cloud documentation refers to `dig` command-line tool so I decided to keep this style :)

Added an example with `ping`. 

Could you please review?

Thank you!

File: guides/v2.2/cloud/env/environments-start.md [hunk 1]
```diff
@@ -226,6 +226,25 @@ If you have a corporate firewall that blocks outgoing SSH connections, you can a
...
+
+Use `ping` command for retrieving IP address for particular Cloud instance.
+
+Example of usage:
+
+```bash
+ping integration-abcd123-abcd78910.us-3.magentosite.cloud
+```
+
+returns
```
Comment (hguthrie) [inline guides/v2.2/cloud/env/environments-start.md]:
```suggestion
Sample response:
```

**Comment by hguthrie** [comment]
Yes, the dig command has been noted in an older topic of the Cloud guide regarding a verification for Fastly caching. It is best to remain as generic as possible when suggesting methods for verification or using only official references when linking to external sites for further information.

File: guides/v2.2/cloud/env/environments-start.md [hunk 2]
```diff
@@ -226,6 +226,25 @@ If you have a corporate firewall that blocks outgoing SSH connections, you can a
 </tr>
 </table>
 
+## Get IP address of Cloud instance
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.2/cloud/env/environments-start.md]: yes, good point

**Comment by atwixfirster** [comment]
> Thanks for the contribution!

With my pleasure!

**Comment by hguthrie** [comment]
running tests

**Pull request merged**